packages/file_selector/file_selector/README.md

```diff
@@ -76,6 +90,15 @@ final XFile textFile =
 await textFile.saveTo(path);
 ```
 
+## Get directory path
+
+<?code-excerpt "get_directory_page.dart (GetDirectory)"?>
+```dart
+const String confirmButtonText = 'Choose';
```

Comment:
It would be better to make a new example snippet for the README; setting the confirm button text is something that will likely be rare, so isn't necessary to show in to a minimal example.